training/src/app/modules/chuck-norris/components/joke-generator/joke-generator.component.scss

```diff
@@ -0,0 +1,92 @@
+@import "../../../../../styles";
+
+.search-style {
+  display: grid;
+  grid-template-columns:  1fr 0.3fr;
+  grid-gap: 70px;
+  background-color: $off-white-color;
+  border-radius: 0.4em;
+  margin: 8%;
+  width: 80%;
+  padding: 3% 3% 1% 3%;
+
+  .key-word-style {
+    display: grid;
+    grid-template-columns:  0.21fr 0.7fr;
+  }
+}
+
+.random-container {
+  display: grid;
+  grid-template-columns:  1fr 1fr 0.7fr;
+  grid-gap: 10px;
+  background-color: $off-white-color;
+  border-radius: 0.4em;
+  margin: 8%;
+  width: 80%;
+  padding: 3% 3% 1% 3%;
+
+  mat-select {
+    background-color: $white-color;
+    height: 40px;
+    border-radius: 5px;
+  }
+
+  .your-name-style {
+    display: grid;
+    grid-gap: 10px;
+    grid-template-columns:  0.25fr 0.9fr;
+  }
+
+  .category-style {
+    display: grid;
+    grid-template-columns:  0.1fr 0.9fr;
+    grid-gap: 10px;
+  }
+}
+
+a {
+  padding-top: 12%;
+}
+
+button {
+  color: $white-color;
+}
+
+.btn-three-random, .btn-three-search {
+  background-color: $orange-color;
+  height: 42px;
+  font-family: 'Roboto', sans-serif;
+}
+
+@media screen and (max-width: 680px) {
+  .random-container {
+    display: block;
+
+    .category-style {
+      display: grid;
+      grid-template-columns:  0.25fr 0.9fr;
+      grid-gap: 10px;
+    }
+
+    button {
+      display: block;
+      margin-left: auto;
+      margin-right: auto;
+      margin-top: 6%;
+      width: 25%;
+    }
+
+  }
+  .search-style {
+    display: block;
+
+    button {
+      display: block;
+      margin-left: auto;
+      margin-right: auto;
+      margin-top: 6%;
+      width: 25%;
```

Comment:
width of a button should be pixel and not percentage. also, it doesnt looks like this style is working